src/styles/_forms.scss

```diff
@@ -1,156 +1,144 @@
 @use 'range-slider' as *;
 @use 'typography';
 
-input,
-select,
-option,
-textarea {
-  width: 100%;
-  padding: 0.5rem;
-  color: var(--color-text);
-  font-weight: 500;
-  font-size: 1rem;
-  background: var(--color-white);
-  border: 1px solid var(--color-text);
-
-  &:disabled {
-    color: var(--color-text);
-    background: var(--color-grayscale-10);
-    cursor: not-allowed;
+/*
+  Some Punkt elements, e.g. inputs, check- and radio-buttons, are still marked
+  work in progress (as per v. 3.1.2). The following is an attempt to match the
+  design in Figma until these are fully implemented in CSS and as components.
+*/
+
+.pkt-form {
+  color: var(--color-blue-dark);
+  line-height: 1.5rem;
+
+  .pkt-hr {
+    margin-top: 0;
   }
 }
 
-.form-group {
-  display: block;
-  margin: 1rem 0;
+.pkt-form-group {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  color: var(--color-blue-dark);
+
+  &--row {
+    flex-direction: row;
+  }
 
   &--error {
-    .form-input, .vs__dropdown-toggle, .vs__dropdown-menu {
-      border-color: var(--color-error);
+    .pkt-form-input,
+    .pkt-form-textarea {
+      border-color: var(--color-red);
     }
-    .form-label {
-      color: var(--color-error);
+    .v-select {
+      --vs-border-color: var(--color-red);
     }
   }
 }
 
-.form-label {
-  display: block;
-  margin-bottom: 0.5rem;
-  color: var(--color-text);
-
-  &--hasPrimaryBackground {
-    color: var(--color-text-secondary);
+.pkt-form-label {
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+  font-weight: 500;
+  // According to Figma, `font-size` and `line-height` for labels should be
+  // `1.125rem` and `1.5rem` respectively. This is for now scaled down somewhat
+  // until the rest of the design is adjusted.
+  font-size: typography.$font-size-2;
+  line-height: 1.5rem;
+
+  .pkt-badge {
+    // Align badges, used to indicate optional form fields, with Figma design.
+    height: auto;
+    padding: 0.125rem 0.5rem;
+    font-weight: 300;
+    font-size: typography.$font-size-1;
+    line-height: 1.375rem !important; // 22px
```

Comment:
Kan vi ikke laste våre komponenter etter Punkt-tingene i `main.scss` for å slippe disse `!important`-ene?